src/regression_tracker.py

```diff
@@ -30,44 +31,71 @@ def _stop(self, sub_id):
         if sub_id:
             self._api_helper.unsubscribe_filters(sub_id)
 
-    def _create_regression(self, failed_node, last_successful_node):
+    def _create_regression(self, failed_node, last_pass_node):
         """Method to create a regression"""
         regression = {}
-        for field in self._regression_fields:
-            regression[field] = failed_node[field]
-
         regression['kind'] = 'regression'
+        regression['name'] = failed_node['name']
+        regression['path'] = failed_node['path']
+        regression['group'] = failed_node['group']
+        regression['state'] = 'done'
         regression['data'] = {
             'fail_node': failed_node['id'],
-            'pass_node': last_successful_node['id'],
+            'pass_node': last_pass_node['id'],
+            'arch': failed_node['data'].get('arch'),
+            'defconfig': failed_node['data'].get('defconfig'),
+            'compiler': failed_node['data'].get('compiler'),
+            'platform': failed_node['data'].get('platform'),
```

Comment:
We can create a new list with `Node.data` fields and use it to assign these values the same way as with `self._regression_fields`.

Reply:
Any reason for creating this alternate PR?